app/components/avatar/avatar.html.erb

```diff
@@ -1,3 +1,3 @@
 <div class="Avatar">
-    <img src="<%= url || 'https://source.unsplash.com/random?cat' %>" />
+    <img src="<%= url %>" alt="" />
```

Comment:
Does it help in any way to have an empty `alt`?

Reply:
Yes, an image tag without an `alt` attribute is invalid HTML. An empty `alt` attribute indicates that the image is optional. This way, many screen readers will simply skip the element. If you completely omit the attribute, screen readers might announce there’s an image element, but aren’t able to provide any details what is shown – which is confusing and unnecessary.